provisioning/containers/gui-common/README.md

```diff
@@ -0,0 +1,22 @@
+# Docker-based GUI
+
+This sub-project aims to provide a Dockerized environment where Docker-based GUI applications can run. This is backed by a set of different containers, each with its own specific functions, that together constitute and offer the Docker-based GUI application.
```

Comment:
This sentence looks ok, but not here. Probably is much better in the (parent) "containers" folder.